Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM rust:1.32.0
 
 RUN cargo install cargo-watch
 
+RUN apt-get update
+RUN apt-get install -y ruby-full
```

Comment:
This should be in one line with a `&&`. There's an explanation for why here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/